Tests/SuperLinq.Test/ToArrayTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿namespace Test;
```

Comment:
Additional test should be added to confirm that exception will be thrown when length is <= 0.

Reply:
I'm throwing exceptions only on length < 0. I prefer to handle 0 length. There is the possibility to provide an empty enumerable and then we expect 0 to be supplied as length. It would be a rare case, but possible if people are calculating length.